sdk/core/System.ClientModel/src/Internal/PipelineMessageSanitizer.cs

```diff
@@ -0,0 +1,157 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
If we've merged the performance enhancements to the Azure.Core sanitizer type, it would be good to update this one with those as well.